app/views/components/heading.scala.html

```diff
@@ -14,11 +14,18 @@
  * limitations under the License.
  *@
 
-@(title: String, heading: String)(implicit messages: Messages)
+@(sectionHeader: Option[String] = None, title: Option[String], hint: Option[String] = None)
 
-<fieldset>
-    <legend class="heading-large" role="heading" aria="" level="1">
-        <span class="heading-secondary">@messages(title)</span>
-        <h1 style="margin:0">@messages(heading)</h1>
-    </legend>
-</fieldset>
\ No newline at end of file
+<header class="page-heading">
+    @if(sectionHeader.nonEmpty) {
+        <span class="visuallyhidden">This section is </span>
+        <span id="section-header" class="heading-secondary">@sectionHeader.get</span>
+    }
+
+    @if(title.nonEmpty) {
+        <h1 id="title" style="margin: 0px">@title.get</h1>
```

Comment:
Is there a `m-0` class you can use instead of the inline style?  If not can we create one please :)

Reply:
This has been addressed.